com.unity.render-pipelines.high-definition/Editor/Material/UIBlocks/SurfaceOptionUIBlock.cs

```diff
@@ -79,6 +79,8 @@ internal static class Styles
 
             public static GUIContent doubleSidedEnableText = new GUIContent("Double-Sided", "When enabled, HDRP renders both faces of the polygons that make up meshes using this Material. Disables backface culling.");
 
+            public static GUIContent customVelocityText = new GUIContent("Add Custom Velocity", "When enabled, HDRP will happen the provided per vertex custom velocity to the motion vector calculation.");            
```

Comment:
I don't fully understand what the tooltip is trying to say. Maybe it should sound something like this: "When enabled, the provided custom velocity is added to the motion vector calculation."?

Reply:
good suggestion